index.js

```diff
@@ -11,23 +11,31 @@ const project = new Project({
     tsConfigFilePath: path.join(src, "tsconfig.json")
 })
 for (const f of project.getSourceFiles("**/*.d.ts")) {
-    const gs = f.getDescendantsOfKind(ts.SyntaxKind.GetAccessor)
-    for (const g of gs) {
-        const s = g.getSetAccessor();
-        g.replaceWithText(`${s ? "" : "readonly "}${g.getName()}: ${g.getReturnTypeNodeOrThrow().getText()}`)
-        if (s) {
-            s.remove()
+    f.forEachDescendant(n => {
```

Comment:
Reverted this change in https://github.com/sandersn/downlevel-dts/pull/4/commits/15f53ae99abd9fb77d80d811a056b0984f792d47 because I wasn't sure this was actually faster. Might be something to consider in the future though and test to see which is faster or use the `SourceFile#traverse` method, which is definitely fastest.